Added details about Pickers only associated to a specific facility #1298 (#1360)
hotwax/oms-documentation: documents/system-admin/administration/users/add-picker.md

1. Navigate to the `user management app` of HotWax Commerce.
2. Open the user you want to make a `Picker` or [create a new user](create-user.md)

> 3. Go to the `Fulfillment card` tab and toggle on `show as a picker`.

Reviewer: ```suggestion
3. Go to the `Fulfillment` card and toggle on `Show as a picker`.
```

> 4. Click the 'Add Facilities' button; this will open a new form. Select the facility for which you want to add the picker.

Reviewer: ```suggestion
4. Click the 'Add Facilities' button and select the facilities the picker is linked to.
```
